tests/console/yast2_lan.pm

```diff
@@ -49,23 +50,24 @@ sub run {
     script_run('ls -alF /etc/sysconfig/network/');
     save_screenshot;
 
+    my $IS_NM;
```

Comment:
- let's keep same variables naming style 
- no point to split variable declaration and init
-  you can achieve the same without using 'if' 
```suggestion
    my $is_nm =! script_run("systemctl is-active NetworkManager");
```

Reply:
I thought Travis complained when we did not initialize the variables, I'm happy to see it's not the case. Thank you for the review !